packages/flutter/lib/src/foundation/debug.dart

```diff
@@ -132,3 +134,31 @@ String? activeDevToolsServerAddress;
 
 /// The uri for the connected vm service protocol.
 String? connectedVmServiceUri;
+
+/// If memory allocations are enabled, dispatch object creation.
+///
+/// This method is not member of FlutterMemoryAllocations, because
+/// [FlutterMemoryAllocations] should not increase size of the Flutter application
+/// if memory allocations are disabled.
+///
+/// Returns true to make it easier to be wrapped into `assert`.
+bool maybeDispatchObjectCreated(String library, String className, Object object) {
```

Comment:
The word "dispatch" (in the function names and in the dartdocs) is confusing. Is there a better word to describe it? Perhaps we are "recording" object creation, or "tracking", or "logging" it?

Reply:
Hmm, yeah, if "dispatch" is already stuck, then I guess following the established convention makes sense. I wish I caught it earlier though. The word "dispatch" has no connection to allocation tracking.

Reply:
It means communicate that something was allocated or released.
The class FlutterMemoryAloocations does not record anything. But you can subscribe to it and listen to allocations.
Does it help?